lambda/css-api/src/routes.ts

```diff
@@ -140,6 +143,66 @@ export const setRoutes = (app: any) => {
     }
   });
 
+  app.get(`/integrations/:integrationId/:environment/logs`, logsRateLimiter, async (req, res) => {
+    /*#swagger.auto = false
```

Comment:
Can we get a comment on whay there's this massibe block of commented out code? Or is that how swagger generates docs?

Reply:
This is being used on all the routes in the css-api, swagger autogenerates from it. Not a huge fan of it but following the existing setup for consistency